src/tests/test_hiv_status.py

```diff
@@ -61,40 +62,45 @@ def test_HIV_risk_vector():
     pop.data[col.SEX_MIX_AGE_GROUP] = np.array(age_group_list)
     pop.data[col.HIV_STATUS] = np.array(HIV_list)
     pop.data[col.NUM_PARTNERS] = 1  # give everyone a single stp to start with
-    
-    # if everyone has the same number of partners, probability of being with someone with HIV should be = HIV prevalence 
+
+    # if everyone has the same number of partners,
+    # probability of being with someone with HIV should be = HIV prevalence
     hiv_module.update_partner_risk_vectors(pop)
-    #print(hiv_module.stp_HIV_rate)
     expectation = np.array([0.1]*5)
     assert np.allclose(hiv_module.stp_HIV_rate[SexType.Male], expectation)
     assert np.allclose(hiv_module.stp_HIV_rate[SexType.Female], expectation)
 
     # Check for differences in male and female rate correctly
-    # change HIV rate in men to double 
+    # change HIV rate in men to double
     males = pop.data.index[pop.data[col.SEX] == SexType.Male]
-    # transform group fails when only grouped by one field --> appears to change the type of the object passed to the function! 
-    male_HIV_status = pop.transform_group([col.SEX_MIX_AGE_GROUP, col.SEX], lambda x, y: np.array([True] * (2 * N_group // HIV_ratio) + [False] * (N_group - 2*N_group // HIV_ratio)), False, males)
+    # transform group fails when only grouped by one field
+    # appears to change the type of the object passed to the function!
```

Comment:
Just noting that this should be fixed as part of #71, so we can update the following lines when merged.